### ISSUE TITLE
Update and publish versions

### ISSUE THREAD

**Issue by OrangeDog** [fyockm/bootstrap-css-only#53]
The latest versions of Bootstrap are 4.6.0 and 5.1.0.

Also, the previous GitHub release was never published to NPM.

**Comment by davecoffin** [comment]
Yes please. Bootstrap 5.x solves a compatibility issue with angular 12 where the :lang selector is depreciated. Any chance for a bump?

**Comment by monis01** [comment]
Yup, looking for version upgrade to 5.x?  anything planned ?
